docs/installation/migrate.rst

```diff
@@ -16,77 +16,130 @@ You can set up continuous migration of |sr| to maintain a hybrid deployment ("ex
 cloud") or "lift and shift" all to |ccloud| using a one-time migration.
 
 Continuous Migration
---------------------
+~~~~~~~~~~~~~~~~~~~~
 
 For continuous migration, you can use your self-managed |sr| as a primary and
 |sr-ccloud| as a secondary. New schemas will be registered directly to the
 self-managed |sr|, and |crep| will continuously copy schemas from it to
 |sr-ccloud|, which is set to IMPORT mode.
 
-**TBD diagram showing continuous migration**
-
 One-time Migration
-------------------
+~~~~~~~~~~~~~~~~~~
 
 Choose a one-time migration to move all data to a fully-managed |ccloud|
 service. In this case, you migrate your existing self-managed |sr| to
 |sr-ccloud| as a primary. All new schemas are registered to |sr-ccloud|. In the
 scenario, there is no migration from |sr-ccloud| back to the self-managed |sr|.
 
-**TBD diagram showing one-time migration**
+Prerequisites
+-------------
 
+The Run Book describes how to perform a |sr| migration applicable to any type of
+deployment (from on-premise or data center to |sr-ccloud|). The examples also
+serve as a quick start tutorial you can use to walk through the basics of
+migrating schemas from a local cluster to |ccloud|. 
+
+Before you begin, verify that you have:
+
+* |ccloud| `5.2` or newer to serve as the destination |sr|
```

Comment:
What does this mean: how do you verify CCloud version?  I didn't know you could see version in CCloud